### PR TITLE
Upgrade agency version used in CI

### PR DESCRIPTION
**Pull request by Patrik-Stas**
Signed-off-by: Patrik Stas <patrik.stas@absa.africa>

### CODE REVIEW

File: .github/actions/code-coverage/action.yml [hunk 1]
```diff
@@ -37,9 +37,31 @@ runs:
         set -x
         sudo rm -rf "/usr/local/share/boost" "$AGENT_TOOLSDIRECTORY" "/usr/local/lib/android" "/usr/share/dotnet"
         if [[ ${{ inputs.run-services }} == true ]]; then
-          docker run --rm -d --name indypool --network host ${{ inputs.pool-img-name }}
-          docker run --rm -d --name postgres --network host -e POSTGRES_PASSWORD=mysecretpassword postgres:12.1
-          if [[ -n "${{ inputs.agency-img-name }}" ]]; then docker run --rm -d --name vcxagency --network host --env-file ci/agency/localhost.env ${{ inputs.agency-img-name }}; fi
+          docker run --rm -d --name indypool  --network host $DOCKER_IMG_NAME_POOL
```
Comment (mirgee) [inline .github/actions/code-coverage/action.yml:40]: Why is this test in code-coverage github action?

File: .github/actions/code-coverage/action.yml [hunk 2]
```diff
@@ -37,9 +37,31 @@ runs:
...
+                    -e MYSQL_DATABASE=agency_application \
+                    -e MYSQL_USER=root \
+                    -e MYSQL_PASSWORD=mysecretpassword \
+                    -e MYSQL_HOST=localhost \
+                    -e MYSQL_PORT=3306 \
+                   "$DOCKER_IMG_NAME_AGENCY" \
+                   bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
+            docker run --rm -d --name vcxagency --network host --env-file ci/agency/localhost.env ${{ inputs.agency-img-name }}
+            sleep 5
+            docker logs vcxagency
```
Comment (mirgee) [inline .github/actions/code-coverage/action.yml:63]: Probably forgot to remove

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -568,9 +568,41 @@ jobs:
...
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
+
+      - name: Start services for integration tests
+        run: |
+          set -x
           docker run --rm -d --name vcxagency --network host --env-file ci/agency/localhost.env $DOCKER_IMG_NAME_AGENCY
+          sleep 5
+          docker logs vcxagency
```
Comment (mirgee) [inline .github/workflows/main.yml:605]: Probably forgot to remove

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -629,10 +661,41 @@ jobs:
...
```
Comment (mirgee) [inline .github/workflows/main.yml:698]: dtto

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -816,9 +911,41 @@ jobs:
...
```
Comment (mirgee) [inline .github/workflows/main.yml:948]: dtto

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -816,9 +911,41 @@ jobs:
...
+                         --network host \
+                          -e MYSQL_DATABASE=agency_application \
+                          -e MYSQL_USER=root \
+                          -e MYSQL_PASSWORD=mysecretpassword \
+                          -e MYSQL_HOST=localhost \
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
+
+      - name: Start services for integration tests
```
Comment (mirgee) [inline .github/workflows/main.yml:943]: Incorrect name

File: .github/workflows/main.yml [hunk 7]
```diff
@@ -689,9 +752,41 @@ jobs:
...
+        run: |
+          set -x
           docker run --rm -d --name vcxagency --network host --env-file ci/agency/localhost.env $DOCKER_IMG_NAME_AGENCY
+          sleep 5
+          docker logs vcxagency
```
Comment (mirgee) [inline .github/workflows/main.yml:789]: dtto

File: .github/workflows/main.yml [hunk 8]
```diff
@@ -568,9 +568,41 @@ jobs:
...
+                          -e MYSQL_USER=root \
+                          -e MYSQL_PASSWORD=mysecretpassword \
+                          -e MYSQL_HOST=localhost \
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
+
+      - name: Start services for integration tests
+        run: |
+          set -x
```
Comment (mirgee) [inline .github/workflows/main.yml:602]: I don't think `set -x` is necessary if running just one command

File: .github/workflows/main.yml [hunk 9]
```diff
@@ -568,9 +568,41 @@ jobs:
...
+                         --network host \
+                          -e MYSQL_DATABASE=agency_application \
+                          -e MYSQL_USER=root \
+                          -e MYSQL_PASSWORD=mysecretpassword \
+                          -e MYSQL_HOST=localhost \
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
+
+      - name: Start services for integration tests
```
Comment (mirgee) [inline .github/workflows/main.yml:600]: Incorrect name (applies elsewhere in this PR)

File: .github/workflows/main.yml [hunk 10]
```diff
@@ -568,9 +568,41 @@ jobs:
...
           set -x
           sudo rm -rf "/usr/local/share/boost" "$AGENT_TOOLSDIRECTORY" "/usr/local/lib/android" "/usr/share/dotnet"
-          docker run --rm -d --name indypool --network host $DOCKER_IMG_NAME_POOL
-          docker run --rm -d --name postgres --network host -e POSTGRES_PASSWORD=mysecretpassword postgres:12.1
+          docker run --rm -d --name indypool  --network host $DOCKER_IMG_NAME_POOL
+          docker run --rm -d --name mysql     --network host -e MYSQL_ROOT_PASSWORD=mysecretpassword mysql:5.7.35
+
+      - name: Create Wallet DB schema, run migrations
+        run: |
+          set -x
```
Comment (mirgee) [inline .github/workflows/main.yml:576]: I don't think `set -x` is necessary when running single command (applies everywhere in this PR)

File: .github/workflows/main.yml [hunk 11]
```diff
@@ -568,9 +568,41 @@ jobs:
...
+          docker run --rm --name vcxagencynode_migrations_data \
+                         --network host \
+                          -e MYSQL_DATABASE=agency_application \
+                          -e MYSQL_USER=root \
+                          -e MYSQL_PASSWORD=mysecretpassword \
+                          -e MYSQL_HOST=localhost \
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
```
Comment (mirgee) [inline .github/workflows/main.yml:598]: These two steps run a container, perform a migration and then proceed to delete it, which does not seem very useful. What am I missing?

File: .github/workflows/main.yml [hunk 12]
```diff
@@ -568,9 +568,41 @@ jobs:
...
-          docker run --rm -d --name indypool --network host $DOCKER_IMG_NAME_POOL
-          docker run --rm -d --name postgres --network host -e POSTGRES_PASSWORD=mysecretpassword postgres:12.1
+          docker run --rm -d --name indypool  --network host $DOCKER_IMG_NAME_POOL
+          docker run --rm -d --name mysql     --network host -e MYSQL_ROOT_PASSWORD=mysecretpassword mysql:5.7.35
+
+      - name: Create Wallet DB schema, run migrations
+        run: |
+          set -x
+          docker run --rm --name vcxagencynode_migrations_wallet \
+                         --network host \
```
Comment (mirgee) [inline .github/workflows/main.yml:578]: Using host network is unnecessary

**Review by mirgee — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 10 above)
Comment (Patrik-Stas) [reply .github/workflows/main.yml]: `set -x` can be useful even if just a single command is to be run, as it gives you ability to see what bash variables been interpolated

File: .github/workflows/main.yml (hunk 7 above)
Comment (Patrik-Stas) [reply .github/workflows/main.yml]: Removed

File: .github/actions/code-coverage/action.yml (hunk 2 above)
Comment (Patrik-Stas) [reply .github/actions/code-coverage/action.yml]: Not really but removed :-D I guess it's reasonable to expect agency and its migrations to run properly in this repo.

File: .github/workflows/main.yml (hunk 10 above)
Comment (mirgee) [reply .github/workflows/main.yml]: The UI shows the executed code and defined environment variables anyway.

**Pull request merged**